diff --git a/RNWorklets.podspec b/RNWorklets.podspec
index 11e10b7a731f754b524c9e8ccd8126334b6adb68..4522a2336de490d960d3edd28fc6fad230d64cae 100644
--- a/RNWorklets.podspec
+++ b/RNWorklets.podspec
@@ -57,7 +57,7 @@ Pod::Spec.new do |s|
 
   s.dependency 'React-jsi'
   s.dependency 'React-hermes'
-  
+
   s.pod_target_xcconfig = {
     "USE_HEADERMAP" => "YES",
     "DEFINES_MODULE" => "YES",
@@ -70,6 +70,7 @@ Pod::Spec.new do |s|
       '"$(PODS_ROOT)/DoubleConversion"',
       '"$(PODS_ROOT)/Headers/Private/React-Core"',
       '"$(PODS_ROOT)/Headers/Private/Yoga"',
+      '"$(PODS_CONFIGURATION_BUILD_DIR)/React-utils/React_utils.framework/Headers/react/utils/platform/ios"',
     ].join(' '),
     "FRAMEWORK_SEARCH_PATHS" => '"${PODS_CONFIGURATION_BUILD_DIR}/React-hermes"',
     "CLANG_CXX_LANGUAGE_STANDARD" => "c++20",
diff --git a/apple/worklets/apple/Networking/WorkletsNetworking.mm b/apple/worklets/apple/Networking/WorkletsNetworking.mm
index 2ea7a712aa399f0b9c073c9696ea4a40058605f3..01852a9f5c387c98f27665400b338321dc1b0f02 100644
--- a/apple/worklets/apple/Networking/WorkletsNetworking.mm
+++ b/apple/worklets/apple/Networking/WorkletsNetworking.mm
@@ -5,9 +5,9 @@
 
 #import <mutex>
 
-#import <FBReactNativeSpec/FBReactNativeSpec.h>
-#import <React-Core/React/RCTFollyConvert.h>
-#import <React-jsi/jsi/JSIDynamic.h>
+#import <ReactCommon/RCTTurboModule.h>
+#import <react/utils/FollyConvert.h>
+#import <jsi/JSIDynamic.h>
 #import <React/RCTAssert.h>
 #import <React/RCTConvert.h>
 #import <React/RCTHTTPRequestHandler.h>
diff --git a/apple/worklets/apple/WorkletsModule.mm b/apple/worklets/apple/WorkletsModule.mm
index 8aabb1432405cc41ab5e6e52e975382c7bc13d16..506516b5e0df0a08fc11ef6e5144697a296452d0 100644
--- a/apple/worklets/apple/WorkletsModule.mm
+++ b/apple/worklets/apple/WorkletsModule.mm
@@ -13,7 +13,6 @@
 #import <React/RCTCallInvoker.h>
 
 #ifdef WORKLETS_FETCH_PREVIEW_ENABLED
-#import <FBReactNativeSpec/FBReactNativeSpec.h>
 #import <React/RCTNetworking.h>
 #import <ReactCommon/RCTTurboModule.h>
 #import <worklets/apple/Networking/WorkletsNetworking.h>
diff --git a/lib/module/bundleMode/network.native.js b/lib/module/bundleMode/network.native.js
index 0e0e386273ee8854b58729dc5398dab0572867dd..97e4cc1ae7bd33886513d08dfae8ca06368a1ce2 100644
--- a/lib/module/bundleMode/network.native.js
+++ b/lib/module/bundleMode/network.native.js
@@ -6,6 +6,8 @@
  * The NetworkingModule itself is injected via C++.
  */
 export function initializeNetworking() {
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
+  require('react-native/Libraries/TurboModule/TurboModuleRegistry');
   const TurboModules = globalThis.TurboModules;
   TurboModules.set('FileReaderModule', makeMockTurboModule('FileReaderModule'));
   TurboModules.set('PlatformConstants', makeMockTurboModule('PlatformConstants'));
diff --git a/src/bundleMode/network.native.ts b/src/bundleMode/network.native.ts
index ec5c64eec8a3ba71b652db5096aa1775d72f70c0..0c9211535fbe394d7522624a75ab2d2ce8c4b12d 100644
--- a/src/bundleMode/network.native.ts
+++ b/src/bundleMode/network.native.ts
@@ -6,6 +6,8 @@
  * The NetworkingModule itself is injected via C++.
  */
 export function initializeNetworking() {
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
+  require('react-native/Libraries/TurboModule/TurboModuleRegistry');
   const TurboModules = globalThis.TurboModules;
 
   TurboModules.set('FileReaderModule', makeMockTurboModule('FileReaderModule'));
